### ISSUE TITLE
Plugin causing bunch of errors in Unity

### ISSUE THREAD

**Issue by dominykasmk** [googlesamples/unity-jar-resolver#276]
The existence of the plugin in a Unity 2019.3+ project causes errors to be thrown constantly in the Console
```
"ArgumentException: Requested value 'X86' was not found.
System.Enum+EnumResult.SetFailure (System.Enum+ParseFailureKind failure, System.String failureMessageID, System.Object failureMessageFormatArgument) (at <ad04dee02e7e4a85a1299c7ee81c79f6>:0)
System.Enum.TryParseEnum (System.Type enumType, System.String value, System.Boolean ignoreCase, System.Enum+EnumResult& parseResult) (at <ad04dee02e7e4a85a1299c7ee81c79f6>:0)
System.Enum.Parse (System.Type enumType, System.String value, System.Boolean ignoreCase) (at <ad04dee02e7e4a85a1299c7ee81c79f6>:0)
System.Enum.Parse (System.Type enumType, System.String value) (at <ad04dee02e7e4a85a1299c7ee81c79f6>:0)
GooglePlayServices.AndroidAbis.EnumValueStringToULong (System.Type enumType, System.String enumValueString) (at /Users/smiles/dev/src/unity-jar-resolver/source/PlayServicesResolver/src/AndroidAbis.cs:182)
GooglePlayServices.AndroidAbis.get_Current () (at /Users/smiles/dev/src/unity-jar-resolver/source/PlayServicesResolver/src/AndroidAbis.cs:241)
GooglePlayServices.PlayServicesResolver.PollAndroidAbis () (at /Users/smiles/dev/src/unity-jar-resolver/source/PlayServicesResolver/src/PlayServicesResolver.cs:1126)
System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at <ad04dee02e7e4a85a1299c7ee81c79f6>:0)
Rethrow as TargetInvocationException: Exception has been thrown by the target of an invocation.
System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at <ad04dee02e7e4a85a1299c7ee81c79f6>:0)
System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) (at <ad04dee02e7e4a85a1299c7ee81c79f6>:0)
System.Delegate.DynamicInvokeImpl (System.Object[] args) (at <ad04dee02e7e4a85a1299c7ee81c79f6>:0)
System.MulticastDelegate.DynamicInvokeImpl (System.Object[] args) (at <ad04dee02e7e4a85a1299c7ee81c79f6>:0)
System.Delegate.DynamicInvoke (System.Object[] args) (at <ad04dee02e7e4a85a1299c7ee81c79f6>:0)
UnityEditor.EditorApplication.Internal_CallUpdateFunctions () (at C:/Projects/Unity3/Editor/Mono/EditorApplication.cs:310)"
```
More info here: https://issuetracker.unity3d.com/product/unity/issues/guid/1178424/

**Comment by markchandler-google** [comment]
Thanks for the bug report @dominykasmk. We are looking into it.

**Comment by stewartmiles** [comment]
@dominykasmk which version of this plugin are you using?  AFAIK this was fixed in `1.2.115` https://github.com/googlesamples/unity-jar-resolver/blob/dc6eb08102d305d8649c0c1449dff6225a50e344/CHANGELOG.md#version-12115---may-28-2019

**Comment by stewartmiles** [comment]
@dominykasmk let us know if this still occurs in the latest release.
